events/EC2/DeleteSnapshot.json

```diff
@@ -9,6 +9,31 @@
     "mitreAttackTechniques": [
         "T1485 - Data Destruction"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1070 - Indicator Removal",
+            "reason": "Deleting snapshots can be part of an effort to remove indicators of compromise or evidence of malicious activity."
+        },
+        {
+            "technique": "T1486 - Data Encrypted for Impact",
+            "reason": "If the adversary has encrypted the data and then deletes snapshots, it makes recovery impossible without the decryption keys, thus increasing the impact."
+        },
+        {
+            "technique": "T1531 - Account Access Removal",
+            "reason": "Deleting snapshots associated with compromised or obsolete accounts can be used to prevent access recovery or hinder incident response efforts."
```

Comment:
Yeah. I placed it as account access emoval just in the vase that this is the only copy remaining for getting access to certain services. I think that it may have been a too extreme edge case. I will remove that technique.